custom-recipes/api-connect/recipe.py

```diff
@@ -35,6 +35,7 @@ def get_partitioning_keys(id_list, dku_flow_variables):
 logger.info("config={}".format(logger.filter_secrets(config)))
 
 credential_parameters = config.get("credential", {})
+behaviour_when_error = config.get("behaviour_when_error", "add-error-column")
```

Comment:
🥜 nitpick: as discussed, maybe we can have a default to `add-error-column` for previously-existing recipes, and `keep-error-column` else